pyproject.toml

```diff
@@ -142,3 +142,25 @@ line-length = 160
 
 [tool.ruff.lint]
 ignore = ["F403", "F405", "E402"]
+
```

Comment:
Can we move all the pixi dust to a different PR? I'd prefer that we keep the PR focused to the advertised change.